Casks/i/impel.rb

```diff
@@ -0,0 +1,30 @@
+cask "impel" do
+  version "1.0.3"
+  sha256 :no_check
+
+  url "https://impel-sparkle-updater.fly.dev/dmg/impel_current.dmg", verified: "impel-sparkle-updater.fly.dev/dmg/impel_current.dmg"
+  name "Impel"
+  desc "Proactive AI assistent for the future"
```

Comment:
Avoid marketing or empty phrases, please.  Website states "An always-on AI companion for your Mac" so the below should be close enough.

```suggestion
  desc "AI Companion"
```